cmake/script/macos_zip.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# Copyright (c) 2024-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit/.
+
+if [ -n "$SOURCE_DATE_EPOCH" ]; then
+  find . -exec touch --date="@$SOURCE_DATE_EPOCH" {} +
```

Comment:
FreeBSD's `touch` does not have `--date`. However both Linux and FreeBSD have `-d YYYY-MM-DDThh:mm:SS[.frac][tz]`

Reply:
Switched from `--date` to `-d` in the recent push.